pkg/printers/humanreadable.go

```diff
@@ -32,10 +32,13 @@ import (
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 )
 
+
+// TableGenerator interface
```

Comment:
I think this comment should be more descriptive. 

Maybe something like
```go
// TableGenerator is an interface for generating metav1beta1.Table provided a runtime.Object
```

Reply:
Fixed.